ktlint.gradle

```diff
@@ -7,7 +7,7 @@ configurations {
 }
 
 dependencies {
-    ktlint depends.kotlin.lint.main
+    ktlint "com.pinterest:ktlint:${versions.ktlint}"
```

Comment:
out of curiosity, what prompted this format change across all deps?

Reply:
Oh, I meant to include that in my PR description ... 

The "monorepo" style is cool for consistency, but means the lint rules for version-out-of-date don't work; the linter can't process our variables. So, seeing as there's a minimum number of projects to update and I wanted to take advantage of the "newer version available" lint check, I decided to flatten this out.